package.json

```
@@ -132,6 +134,7 @@
     "nodemon": "^2.0.4",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
+    "otpauth": "^6.0.4",
```

```suggestion
    "otpauth": "6.0.4",
```

We use fixed dependencies.